conftest.py

```diff
@@ -24,6 +22,8 @@ def pytest_configure(config):
     PYTEST_HEADER_MODULES['Scikit-image'] = 'skimage'
     PYTEST_HEADER_MODULES['asdf'] = 'asdf'
     PYTEST_HEADER_MODULES['pyarrow'] = 'pyarrow'
+
+    from astropy import __version__
```

Comment:
I thought I tried this in an earlier attempt and I didn't work. But if it works now, then this is definitely cleaner.